helpers/calendar/analytics.py

```diff
@@ -22,7 +22,7 @@ class RoomStatistics(graphene.ObjectType):
     has_events = graphene.Boolean()
 
 
-class RoomDailyDurations(graphene.ObjectType):
+class RoomMeetingDurations(graphene.ObjectType):
```

Comment:
Good work @kzyGit. I suggest you condense `RoomMeetingDurations` class to `RoomStatistics` by adding `total_duration` as an attribute.

Reply:
Thanks for the feedback @sam-karis . I have resolved it already